<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1481</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>1051</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1051</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 185, 185);
border-top-left-radius:30px;
border-top-right-radius:30px;
border-bottom:2px solid</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>0</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>武术动作自动评分系统</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>20</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton {
    border: none;
}

#pushButton {

	background-color: rgb(185, 185, 185);
}
#pushButton:hover{

	
	background-color: rgb(232, 232, 232);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="source/pic.qrc">
        <normaloff>:/pic/关闭.svg</normaloff>:/pic/关闭.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>321</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>171</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 185, 185);

border-bottom-left-radius:30px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>16</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>加载视频</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>163</y>
        <width>98</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>预览</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>113</y>
        <width>98</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>163</y>
        <width>33</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pic/play.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>158</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pic/pause.svg);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>151</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid black;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>299</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>107</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pic/pause.svg);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>112</y>
        <width>33</width>
        <height>33</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/pic/play.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>360</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>保存日志</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>392</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>保存图片</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>465</y>
        <width>112</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>模型1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>466</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>模型2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>37</x>
        <y>424</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>骨架可视化</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>216</y>
        <width>98</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>展示最佳帧</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>61</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>加载规则</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>263</y>
        <width>98</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
    alternate-background-color: rgb(212, 38, 255);
    gridline-color: rgb(255, 249, 235);
    border-radius: 8px;
	border:2px solid;
}



QPushButton:hover{
background-color:#d5d5d5;/* 设置按钮被点击时的背景颜色 */
padding:2px;

}
QPushButton:pressed {
    background-color: #a7a7a7; /* 设置按钮被点击时的背景颜色 */
}
</string>
      </property>
      <property name="text">
       <string>清空输出</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>70</y>
       <width>441</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 232, 232);
border-right:1px solid;
border-bottom:1px solid</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>359</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">      QSlider::groove:horizontal {
            border: none;
            height: 2px;
            background-color: lightgray;
        }

        QSlider::handle:horizontal {
            background-color: #f1964c;
            width: 16px;
            margin: -8px 0px -8px 0px;
            border-radius: 8px;

        }

        QSlider::sub-page:horizontal {
            background-color: #f1964c;
        }

</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(0, 0, 0);
color: rgb(113, 113, 113);</string>
      </property>
      <property name="text">
       <string>输入视频显示</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>613</x>
       <y>70</y>
       <width>440</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 232, 232);

border-bottom:1px solid</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: rgb(113, 113, 113);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>输出视频显示</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>449</y>
       <width>891</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_5 {
    background-color: rgb(255, 255, 255);
}

#frame_5 {
    border-bottom-right-radius: 30px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">      QProgressBar {
            border: 2px solid #b9b9b9;
            background-color: white;
            padding-left: 2px;
            padding-right: 2px;
          

        }

        QProgressBar::chunk {
            background-color: #b9b9b9;
            margin-top: 2px;
            margin-bottom: 2px;
        }
  border-top-left-radius:30px;</string>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>731</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #aaaaaa;

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>输出</string>
      </property>
     </widget>
    </widget>
    <zorder>frame_5</zorder>
    <zorder>frame_2</zorder>
    <zorder>frame_3</zorder>
    <zorder>frame_4</zorder>
    <zorder>frame_6</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1054</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
